src/pykeen/models/multimodal/complex_literal.py

```diff
@@ -22,11 +22,7 @@ class ComplExLiteral(MultimodalModel):
     #: The default strategy for optimizing the model's hyper-parameters
     hpo_default = dict(
         embedding_dim=dict(type=int, low=50, high=300, q=50),
-        input_dropout={
-            'type': float,
-            'low': 0.1,
-            'high': 0.3,
-        },
+        input_dropout=dict(type=float, low=0.0, high=0.5, q=0.1)
```

Comment:
Refactor this to a constant `default_dropout_hpo_range`

Reply:
@mberr I'm not sure if I follow - this name has to correspond to the name of the variable in arguments of the `__init__()` of the model

Reply:
I was referring to the RHS, `dict(type=float, low=0.0, high=0.5, q=0.1)`, since it is the same for all locations where dropout HPO ranges are defined.

Reply:
aha. yes, saving it as some constant above would make this more meaningful

Reply:
> I was referring to the RHS, `dict(type=float, low=0.0, high=0.5, q=0.1)`, since it is the same for all locations where dropout HPO ranges are defined.

Should we define it in base.py ?

Reply:
`pykeen.hpo.spaces` or `pykeen.hpo.search_spaces`

Reply:
Done in 497127d921fb84d407bf8a4e25cf79bd23a3a495.